### ISSUE TITLE
Resizing the scan area

### ISSUE THREAD

**Issue by ghost** [wu9007/qrcode_scanner#52]
Is there any way to resize the scan area? I've attempted to edit the second activity xml file, but unsure of what i should change.
Thank you in advance.

**Comment by webberlin0933** [comment]
I have the same problem and I don’t have any idea to solve this problem...

**Comment by rymesaint** [comment]
yeah i need to resize the scan area too, because this one have a smaller area to scan

**Comment by Rijadi02** [comment]
android->app->src->main->res->layout->my_camera.xml

 app:inner_width="200dp"
 app:inner_height="200dp"

You can change these 2 values

**Comment by wu9007** [comment]
Not planned.

qrscan stays a one-shot `scan()`: open a full-screen camera, first decode wins, return `String?`, close. We will not add a customizable scan window / overlay API.

If you need an embedded preview, scan area, overlay, or continuous scan, use [mobile_scanner](https://pub.dev/packages/mobile_scanner). That is the widget; this plugin is not competing with it.

Closing as not planned.

**Comment by wu9007** [comment]
@ — you already looked at `activity_second.xml`. 0.4.0 overlay is `ScanOverlayView` on top of CameraX `PreviewView`. There is still no Dart API for scan-area size, and forking the XML inside `.pub-cache` will be overwritten.

qrscan stays a fixed full-screen one-shot. Use `mobile_scanner` if you need a custom cutout. Closing as not planned.
